### ISSUE TITLE
Just wondering

### ISSUE THREAD

**Issue by tryhardsmurfing** [runeliteplusplus/RuneLitePlusPlus#20]
How you got the animation ID's for hydra attacks even though the runelite devs disabled the animation ID's. Trying to learn as i go :)

Edit: it seems the hydra plugin only works on small ones at the moment, is it because there is no longer an animation id on the Boss version? Is there a workaround for that?

**Comment by supermxn** [comment]
RL did block some stuff, I've talked to someone he said that you will have to "unblock" those things by modifying the RL client for it to read it and it is not an easy process from what he said. Hopefully someone will message or assist.

**Comment by tryhardsmurfing** [comment]
Yeah, im sure theres a way to get the animation - im not sure if they still have the hooks that grab the animation and theyre hiding the result, or if they removed the hooks completely. 

There must be some other way aswell thats more complicated. You could write your own animation class and use the npc model type methods to identify a difference between when it ranges and when it mages and use that as the basis.

**Comment by xKylee** [comment]
You should be able to add them into Runelite-api/src/main/java/net/runelite/api/AnimationID.java

Vorkath for instance:

	// Vorkath
	public static final int VORKATH_ATTACK = 7952;
	public static final int VORKATH_FIRE_BOMB_ATTACK = 7960;
	public static final int VORKATH_ACID_ATTACK = 7957;


Then just associate the two?

**Comment by runeliteplusplus** [comment]
RuneLite blocks a lot of features these days. I'm assuming they watch this github as both clan member attack option disabling and vorkath were patched very quickly.

They are blocking these functions as part of the hidden code they run ever since they went closed source. It is possible to work around this as others have mentioned, but at that point you aren't running RuneLite, you're running a fork, a client for a client, and it isn't possible for others to verify that it is safe.

If they are watching for new plugins and patching them as they're made, there is little left for a RL client such as this I'm afraid.

**Comment by supermxn** [comment]
I think you should go closed source, by making a discord or whatnot?

**Comment by xKylee** [comment]
> I think you should go closed source, by making a discord or whatnot?

agreed

**Comment by tryhardsmurfing** [comment]
I agree with the commenters above - it would be cool if you maybe started a discord and taught us how to go around the runelite impositions on things like this just for our own personal clients and more than anything for knowledge, even though it means we cant make a verifiable runelite client.

Edit:

While im here, ive been trying to recreate the tob plugins for fun and got them working basically perfectly but ive experienced this bug twice now and i dont know what causes it (or how i fixed it the first time).

https://imgur.com/FjIE18Z

If you look bottom right, you can see the highlighted tiles that are meant to be on the blood spawns in the maiden room are out there in the blackness, anyone got any experience with something like this? cheers

**Comment by runeliteplusplus** [comment]
> I agree with the commenters above - it would be cool if you maybe started a discord and taught us how to go around the runelite impositions on things like this just for our own personal clients and more than anything for knowledge, even though it means we cant make a verifiable runelite client.
> 
> Edit:
> 
> While im here, ive been trying to recreate the tob plugins for fun and got them working basically perfectly but ive experienced this bug twice now and i dont know what causes it (or how i fixed it the first time).
> 
> https://imgur.com/FjIE18Z
> 
> If you look bottom right, you can see the highlighted tiles that are meant to be on the blood spawns in the maiden room are out there in the blackness, anyone got any experience with something like this? cheers

A discord is maybe a good idea, but assuming RuneLite watches this github, how can I make a discord without them joining it as well?

Are you using word or local point for those tile markings? Somewhere you are getting / setting the wrong data for the tile highlight. When I get something like that happening I print out all relevant data and see what could be going wrong.

**Comment by supermxn** [comment]
make a discord, then you make that text section for a certain role only, for example "verified members" can only see/talk in the general chat, so without it. they can only see the section like news, faqs etc if you have. add me on discord and i will be happy to help
discord: Superman#0001

**Comment by xKylee** [comment]
Discord would be good to create a central repository out of RL’s eyes - could even make the git private and access to those only who fit the verified role. I’m down for a discord group

**Comment by supermxn** [comment]
> 
> 
> Discord would be good to create a central repository out of RL’s eyes - could even make the git private and access to those only who fit the verified role. I’m down for a discord group

add me on it 😃

**Comment by deliveryman9** [comment]
Best way to go about this in my opinion is stop outputting a .jar file for anyone to run. That is catching the eye of Runelite. It makes it just as easy to run as the standard client. This could also cause a Jagex response to Runelite and its open source nature. Plugins by themselves will still catch the eyes of the devs, but not nearly the same attention as a completed client will. 

Discord works to a certain extent, but as you said it will get spied on. Private repo could work as well, but you must have pro to have more than 5 viewers. I really appreciate the plugins, but a completed client definitely pulls unnecessary attention to the project.

**Comment by tryhardsmurfing** [comment]
> > I agree with the commenters above - it would be cool if you maybe started a discord and taught us how to go around the runelite impositions on things like this just for our own personal clients and more than anything for knowledge, even though it means we cant make a verifiable runelite client.
> > Edit:
> > While im here, ive been trying to recreate the tob plugins for fun and got them working basically perfectly but ive experienced this bug twice now and i dont know what causes it (or how i fixed it the first time).
> > https://imgur.com/FjIE18Z
> > If you look bottom right, you can see the highlighted tiles that are meant to be on the blood spawns in the maiden room are out there in the blackness, anyone got any experience with something like this? cheers
> 
> A discord is maybe a good idea, but assuming RuneLite watches this github, how can I make a discord without them joining it as well?
> 
> Are you using word or local point for those tile markings? Somewhere you are getting / setting the wrong data for the tile highlight. When I get something like that happening I print out all relevant data and see what could be going wrong.

I've tried by referencing world point and local point, both give same result. I printed out the worldpoint that is being set in the code and if i use the dev tool tile location and hover the tile in game, it confirms it is selecting the correct tile via worldpoint - yet the highlights are still off in the distance in blackness? 

This only happens to me by the way, I have sent my client to a friend and it works perfectly fine for him.

Edit: I made a new user on my own computer and tested running the client on there - it works perfectly. Why would this be? The same code works for one user and not another..

**Comment by supermxn** [comment]
> 
> 
> > > I agree with the commenters above - it would be cool if you maybe started a discord and taught us how to go around the runelite impositions on things like this just for our own personal clients and more than anything for knowledge, even though it means we cant make a verifiable runelite client.
> > > Edit:
> > > While im here, ive been trying to recreate the tob plugins for fun and got them working basically perfectly but ive experienced this bug twice now and i dont know what causes it (or how i fixed it the first time).
> > > https://imgur.com/FjIE18Z
> > > If you look bottom right, you can see the highlighted tiles that are meant to be on the blood spawns in the maiden room are out there in the blackness, anyone got any experience with something like this? cheers
> > 
> > 
> > A discord is maybe a good idea, but assuming RuneLite watches this github, how can I make a discord without them joining it as well?
> > Are you using word or local point for those tile markings? Somewhere you are getting / setting the wrong data for the tile highlight. When I get something like that happening I print out all relevant data and see what could be going wrong.
> 
> I've tried by referencing world point and local point, both give same result. I printed out the worldpoint that is being set in the code and if i use the dev tool tile location and hover the tile in game, it confirms it is selecting the correct tile via worldpoint - yet the highlights are still off in the distance in blackness?
> 
> This only happens to me by the way, I have sent my client to a friend and it works perfectly fine for him.

you happen to have discord? I may need some assistance for the AOE plugin and as well as try the TOB plugin o.o
Discord: Superman#0001

**Comment by supermxn** [comment]
hello, i've made a discord, with just 3 roles as if now; those are mod, contributor and approved.
for now, you can join it here: https://discord.gg/rvPD6H9 after you have join PM link me your github profile at Superman#0001. Also @runeliteplusplus send me your discord, i will be happy to give you permissions!

**Comment by DoughBoiKush** [comment]
I think we should definetely get a Discord going, that would be step #1 to deciding what direction you should go with this project. I've noticed more than a couple of people (including myself) interested in helping to keep this project going. I've reached out to you about the community Discord and the direction of this project in the past, but was very busy with other things & wasn't able to follow up as much as I would have liked too.

Karma#6992 is my Discord and I've made a server that anyone is welcome to join, but especially anyone that is interested in helping keep this project going & could help w/ development or anything else. There's alot of potential &  demand for a actively updated custom client for OSRS, the one that is used by the PK community doesn't even scratch the surface, & seems like it is completely ignored by Jagex as if it's not there. Just from the people that i personally know whom use this custom RuneLite build, every single one of them are more than interested in a forked client, or better yet a stand a lone client. 


Karma#6992

https://discord.gg/m8kMzwf  <- Discord Server

Let's get together and see where this could go

**Comment by Enza-Denino** [comment]
it looks like the original maintainer doesn't have enough time to keep the plugins up to date or pull in new features

https://github.com/runelite-extended/runelite looks like it is more regularly maintained and updated
